phases/codesign.go

```diff
@@ -1,9 +1,158 @@
 package phases
 
-import "fmt"
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/bitrise-io/bitrise/models"
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/go-yaml/yaml"
+)
+
+// CodesignResult ...
+type CodesignResult struct {
+	AndroidCodesign                                      bool
```

Comment:
are these bool properties needed? at the end, we should just make sure the selected codesign files are uploaded